mne/defaults.py

```diff
@@ -11,23 +11,24 @@
                ref_meg='steelblue', misc='k', stim='k', resp='k', chpi='k',
                exci='k', ias='k', syst='k', seeg='saddlebrown', dipole='k',
                gof='k', bio='k', ecog='k', hbo='#AA3377', hbr='b',
-               fnirs_raw='k', fnirs_od='k', csd='k'),
+               fnirs_raw='k', fnirs_od='k', fnirs_ac='k', fnirs_ph='k', csd='k'),
     units=dict(mag='fT', grad='fT/cm', eeg='µV', eog='µV', ecg='µV', emg='µV',
                misc='AU', seeg='mV', dipole='nAm', gof='GOF', bio='µV',
                ecog='µV', hbo='µM', hbr='µM', ref_meg='fT', fnirs_raw='V',
-               fnirs_od='V', csd='V/m²'),
+               fnirs_od='V', fnirs_ac='V', fnirs_ph=u'\N{DEGREE SIGN}', csd='V/m²'),
     # scalings for the units
     scalings=dict(mag=1e15, grad=1e13, eeg=1e6, eog=1e6, emg=1e6, ecg=1e6,
                   misc=1.0, seeg=1e3, dipole=1e9, gof=1.0, bio=1e6, ecog=1e6,
                   hbo=1e6, hbr=1e6, ref_meg=1e15, fnirs_raw=1.0, fnirs_od=1.0,
-                  csd=1e5),
+                  fnirs_ac=1.0, fnirs_ph=1.0, csd=1e5),
```

Comment:
i wonder if we can just use the fnirs_raw type for the fnirs_AC data to not complicate things

Reply:
I imagine so since all the functions are meant to take in light intensity. We can probably keep it as fnirs_raw until we have reason to change it.

Reply:
great